### DISCUSSION TITLE
Configuration for downloading a subreddit instead of a username?

### DISCUSSION THREAD

**Discussion by wrkcw** [AAndyProgram/SCrawler#121] [Q&A] [answered]
Is there a config for crawling a specific subreddit, rather than a specific reddit user? Thanks!

**Comment by AAndyProgram** [comment, marked as answer]
Of course. Just add the subreddit URL and download. It's simple.
For example, you add a user using the URL `https://reddit.com/user/UserName`. Use the URL `https://reddit.com/r/SubredditName` to add the subreddit.

**Reply by wrkcw** [reply]
Works perfectly, thank you

**Reply by AAndyProgram** [reply]
You're welcome! 😊
